src/components/Characters/Character.css

```diff
@@ -1,31 +1,102 @@
-.Character-wrapper__images {
+.character-wrapper__images {
     border-radius: 10px;
 }
 
-.Character__image{
-    border-radius: 16px;
+.character__image{
```

Comment:
space is missing, ensure ESLint works everywhere

Reply:
done